index.html

```diff
@@ -140,6 +140,7 @@ <h2 class="hide-mobile contact-me-title">Let's chat!</h2>
         <form id="contact-form" action="https://formspree.io/f/xjvlpwgy" method="post">
           <input type="text" name="name" id="contact-me-name" maxlength="30" placeholder="Full name" required>
           <input type="email" name="email" id="contact-me-email" placeholder="Input your email" value="example@email.com" required>
+          <p id="email-error" class = "error-message">Your email should be written in lower case.</p>
```

Comment:
- [ ] Kindly be noticed that the error message should show up close to submit button so kindly put it before the submit button, not before the textarea.